Makefile

```diff
@@ -56,6 +67,8 @@ import: package
 		$(IMPORT_OPTIONS) \
 		$(EXTRA_GRAVITY_OPTIONS) \
 		--include=resources --include=registry .
+	$(call replace,"s/$(VERSION)/0.1.0/g",resources/charts/nethealth/Chart.yaml)
```

Comment:
Why does it get replaced with `0.1.0`?
From the flow it looks as if the replacements are only done for the duration of the `import` target since it operates directly on the source files and guessing the changes are restored to avoid polluting the repository.
Why not `app import` from a temporary location? How many files/much data does it need?

Reply:
It needs only content of resources directory

Reply:
It does not have to be this `resources` directory - you could make a copy and app import from it, no?